test/collectionUtils/InlineValueSetTypeTransformer.test.ts

```diff
@@ -0,0 +1,62 @@
+import InlineValueSetTypeTransformer from "../../src/collectionUtils/InlineValueSetTypeTransformer";
+import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
+import EntityDefinitionBuilder from "../model/dataTypes/EntityDefinitionBuilder";
+import DataType from "../../src/model/dataTypes/DataType";
+import EntityImports from "../../src/model/dataTypes/EntityImports";
+import MemberVariable from "../../src/model/dataTypes/MemberVariable";
+import FilePath from "../../src/model/dataTypes/FilePath";
+
+describe("InlineValueSetTypeTransformer", () => {
+  let transformer: InlineValueSetTypeTransformer;
+  let valueSetEntity: EntityDefinition;
+  let entityType: DataType;
+
+  beforeEach(() => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5963561)